### DISCUSSION TITLE
New Developers

### DISCUSSION THREAD

**Discussion by drewcdisc** [cdisc-org/cdisc-rules-engine#453] [Norms, process, standards]
We want to start a conversation where we will explain things a new developer needs to understand. Here are some example things we might start with:

- Tools and environments
- How are we creating and approving new code (GitHub PRs)
- How are we using GitHub projects for tracking our work
- Understanding acceptance criteria, especially if a dev is new to the domain
- How is our Scrum process

Please add to this list so we can start expanding this.

**Comment by SFJohnson24** [comment]
-  high level overview--both within and around CORE engine.
![image](https://github.com/user-attachments/assets/17641478-7ccf-48ec-9c02-6ce67535a9b3)
![image](https://github.com/user-attachments/assets/6519c35c-55e9-4edb-956b-71cde3122f9c)
these are mock-ups of both the azure architecture and engine architecture functionality from my meetings with Ramil

- domain knowledge specifically around CDISC standards.  
  - the reference guide: https://cdisc-org.github.io/conformance-rules-editor/#/
  - video learning tutorials: https://learn.cdisc.org/index.php?redirect=0
  - various PDF standard guidelines that help guide engineering decisions: https://www.cdisc.org/system/files/members/standard/foundational/SDTMIG%20v3.4-FINAL_2022-07-21.pdf for example

- re:tools and environments: 

  -  SaaS universal viewer for XPT files, 
  - vscode extensions like XML language support by red hat, vscode-pydata-viewer for pkl files, etc.
